C/casacore/build_tarballs.jl

```diff
@@ -0,0 +1,121 @@
+
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "casacore"
+version = v"3.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/casacore/casacore.git", "a735bf5f31ea012ac4f7f7378a1f89c1fc136a06"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/casacore
+
+# Apply upstream patch that fixes i386 builds
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/0001-Fix-compilation-of-RefTable-on-i386-platform-1129.patch
+
+# Apply strerror patch
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/0002-Use-overloading-to-detect-strerror-variant.patch
+
+mkdir build && cd build
+CMAKE_FLAGS=()
+
+# Release build for best performance
+CMAKE_FLAGS+=(-DCMAKE_BUILD_TYPE=Release)
+
+# Install things into $prefix
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
+
+# Explicitly use our cmake toolchain file and tell CMake we're cross-compiling
+CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN})
+CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING:BOOL=ON)
+
+# Disable OpenMP, it'll probably interfere with Julia's threads
+CMAKE_FLAGS+=(-DUSE_OPENMP=OFF)
+
+# Don't build python hooks
+CMAKE_FLAGS+=(-DBUILD_PYTHON=no)
+
+# Stock FFTW3 has threads already baked in
+CMAKE_FLAGS+=(-DFFTW3_DISABLE_THREADS=yes)
+
+# linux builds need to link against rt (they don't do this for some reason)
+if [[ ${target} == *-linux-* ]]; then
+    CMAKE_FLAGS+=(-DCASACORE_ARCH_LIBS="-lrt")
+fi
+
+# Congire
+cmake ${CMAKE_FLAGS[@]} ..
+
+# Make and install
+make -j${nproc}
+make install
+exit
+"""
+
+# Exclude windows, casacore needs pread and pwrite, which are POSIX-only
+# Also exclude FreeBSD, lots of upstream bugs that seem not worth fixing ourself
+platforms = supported_platforms(experimental=true, exclude=(platform)-> Sys.iswindows(platform) || Sys.isfreebsd(platform))
```

Comment:
`experimental=true` does nothing
```suggestion
platforms = supported_platforms(exclude=(platform)-> Sys.iswindows(platform) || Sys.isfreebsd(platform))
```